test/helpdesk_tests/genbank_up_down_duplicate_test.py

```diff
@@ -0,0 +1,170 @@
+import os
```

Comment:
Maybe give a link to the JIRA thread in this module to provide better context

Reply:
added comment